app/app.js

```diff
@@ -28,4 +32,9 @@ app.use((err, req, res, next) => {
   }
 });
 
+app.use((err, req, res, next) => {
+  console.log(err);
+  res.status();
+});
```

Comment:
what is this achieving? Please make your error handler send something useful. this ideally should be the default 500 internal server err